translations/en-us.yaml

```diff
@@ -2875,6 +2922,12 @@ newExternalService:
   header: Add External Service
   alert: "Note: This will only be used if the external service is a target of a load balancer."
 
+newPassword:
+  placeholder: New Password
+  password: New Password*
```

Comment:
* in the template, not the translation (so we don't have to update every language with whether a field is required or not)